demo/clusters/gke/create-cluster.sh

```diff
@@ -35,6 +35,7 @@ DRIVER_NAME=$(from_versions_mk "DRIVER_NAME")
 
 NETWORK_NAME="${DRIVER_NAME}-net"
 CLUSTER_NAME="${DRIVER_NAME}-cluster"
+NODE_VERSION="1.31.1-gke.2105000"
```

Comment:
I would recommend we don't specify the full version `1.31.1-gke.2105000"`, as `2105000` will not be available forever. Instead just specifying `v1.31.1` will default to the latest 1.31 GKE version

Reply:
Ack. Updated.